build.sbt

```diff
@@ -1,16 +1,46 @@
-organization in Global := "com.iheart"
 
-name := "kanaloa"
 
-lazy val root = (project in file(".")).configs(Testing.Integration)
+val commonSettings = Seq(
+  scalaVersion := "2.11.8",
+  scalacOptions ++= Seq(
+    "-encoding", "UTF-8",
+    "-deprecation", // warning and location for usages of deprecated APIs
+    "-feature", // warning and location for usages of features that should be imported explicitly
+    "-unchecked", // additional warnings where generated code depends on assumptions
+    "-Xlint", // recommended additional warnings
+    "-Ywarn-adapted-args", // Warn if an argument list is modified to match the receiver
+    "-Ywarn-inaccessible",
+    "-Ywarn-dead-code",
+    "-Ywarn-numeric-widen"
+  ),
+  organization in Global := "com.iheart",
+  name := "kanaloa"
+)
+
+
+val noPublishing = Seq(publish := (), publishLocal := (), publishArtifact := false)
+
+
+lazy val root = project.in(file("."))
+  .aggregate(core, cluster)
+  .settings(moduleName := "root")
+  .settings(noPublishing)
+
+lazy val core = project
+  .configs(Testing.Integration)
```

Comment:
Should all these dependencies, since they are identical amongst both projects be pulled either into the commonSettings, or into a separate dependenciesSetting?